src/platform/qt/SettingsView.cpp

```diff
@@ -563,8 +563,8 @@ void SettingsView::reloadConfig() {
 	loadSetting("gb.bios", m_ui.gbBios);
 	loadSetting("gbc.bios", m_ui.gbcBios);
 	loadSetting("sgb.bios", m_ui.sgbBios);
-	loadSetting("sgb.borders", m_ui.sgbBorders, true);
-	loadSetting("useCgbColors", m_ui.useCgbColors, true);
```

Comment:
This should not be removed; it's for setting the default when no option is present and entirely unrelated to the bug in question. Why did you remove it?